certify_test.go

```diff
@@ -146,16 +146,18 @@ var _ = Describe("Certify", func() {
 			case 1:
 				// First call is GetCertificate
 				Expect(in3).To(PointTo(MatchAllFields(Fields{
-					"SubjectAlternativeNames":   Equal(append(cli.CertConfig.SubjectAlternativeNames, serverName, cli.CommonName)),
-					"IPSubjectAlternativeNames": Equal(cli.CertConfig.IPSubjectAlternativeNames),
-					"KeyGenerator":              Not(BeNil()),
+					"SubjectAlternativeNames":    Equal(append(cli.CertConfig.SubjectAlternativeNames, serverName, cli.CommonName)),
+					"IPSubjectAlternativeNames":  Equal(cli.CertConfig.IPSubjectAlternativeNames),
+					"URISubjectAlternativeNames": Equal(cli.CertConfig.URISubjectAlternativeNames),
```

Comment:
Could you also add some URI's to the `CertConfig` that runs this test, so we get an end-to-end test of this?

Reply:
Done